.gitignore

```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# just ignoring packages from i downloaded
+package-lock.json
```

Comment:
I don't recommend ignoring `package-lock.json`.

Here's why:
>"The goal of package-lock.json file is to keep track of the exact version of every package that is installed so that a product is 100% reproducible in the same way even if packages are updated by their maintainers."

If you use *npm install* it will only care about packages inside `package.json`, but each npm package depends on another package leading to another and so on. And what keeps track of all the files is `package-lock.json`, so if you install the same project every few months, it will break.

See more [here](https://nodejs.dev/learn/the-package-lock-json-file#:~:text=The%20goal,^%20and%20minor%20releases)

Reply:
You're right!